src/legacy/core_plugins/kibana/public/discover/index.html

```diff
@@ -7,16 +7,22 @@
       <div data-transclude-slot="topLeftCorner" class="kuiLocalBreadcrumbs">
         <h1 tabindex="0" id="kui_local_breadcrumb" class="kuiLocalBreadcrumb" ng-if="opts.savedSearch.id">
           <span class="kuiLocalBreadcrumb__emphasis">
-            <span
-              id="reload_saved_search"
-              aria-label="{{::'kbn.discover.reloadSavedSearchAriaLabel' | i18n: {defaultMessage: 'Reload saved search'} }}"
-              tooltip="{{::'kbn.discover.reloadSavedSearchTooltip' | i18n: {defaultMessage: 'Reload saved search'} }}"
-              tooltip-placement="right"
-              tooltip-append-to-body="1"
-              ng-click="resetQuery()"
-              kbn-accessible-click
-              class="kuiIcon fa-undo small"
-            ></span>&nbsp;
+            <span class="euiFlexItem euiFlexItem--flexGrowZero">
+              <button
+                class="euiButton euiButton--primary euiButton--small euiButtonToggle euiButtonToggle--isIconOnly"
```

Comment:
Let's not force EUI classes in here if possible. When looking at the original issue it honestly just needs the basic `<button>` element (as you have it here) and an actual text label of "Reload" and make it a link color like so:

<img width="228" alt="Screen Shot 2019-06-04 at 10 09 08 AM" src="https://user-images.githubusercontent.com/549577/58885810-d6f11280-86b0-11e9-9eba-0542bb0b825f.png">

Reply:
that's a good catch, thanks a lot!

Reply:
@cchaos, did I get it right that you suggest that I should use inline styles to set `#006BB4` color instead of grabbing it from the framework?

Reply:
No, please don't use inline styles with hard-coded hex colors. Since this is still Angular, you can just use the class `kuiLink` to get the correct styling/coloring.

Reply:
<img width="175" alt="Capture" src="https://user-images.githubusercontent.com/1464331/58914594-9a143280-8727-11e9-8a70-8a5b04e10db0.PNG">

it works and looks good, thanks!